src/spatialdata/_core/spatialdata.py

```diff
@@ -1210,24 +1211,8 @@ def bounding_box(
         """
         Perform a bounding box query on the SpatialData object.
 
-        Parameters
-        ----------
-        axes
-            The axes `min_coordinate` and `max_coordinate` refer to.
-        min_coordinate
-            The minimum coordinates of the bounding box.
-        max_coordinate
-            The maximum coordinates of the bounding box.
-        target_coordinate_system
-            The coordinate system the bounding box is defined in.
-        filter_table
-            If `True`, the table is filtered to only contain rows that are annotating regions
-            contained within the bounding box.
-
-        Returns
-        -------
-        The SpatialData object containing the requested data.
-        Elements with no valid data are omitted.
+        Please see
+        :func:`spatialdata.bounding_box_query` for the complete docstring.
```

Comment:
what is the reason for this change? so you don't have to maintain two docstrings? I think it's fine, but I'm curious

Reply:
Exactly, I would like to have it only in one place because it's easy to forget to update the other (happened). Is there any Sphinx trick we can use here? Asking also @giovp. For this PR I'll keep it like this because I'd like to merge some other PRs dependent on this.

Reply:
I see also torch has something similar to what is in this PR: https://pytorch.org/docs/stable/generated/torch.Tensor.sum.html